main.qml

```diff
@@ -1181,7 +1181,7 @@ ApplicationWindow {
     }
 
     function fiatApiJsonReceived(url, resp, error) {
-        if (error) {
+        if (error) {
```

Comment:
Small thing I noticed: appears that this only removes carriage return. Is this intended? Other source documents like src/qt/downloader.cpp/h are full of carriage return.

Reply:
> appears that this only removes carriage return. Is this intended?

Nope, did it accidentally. But... let's fix it anyway.

> src/qt/downloader.cpp/h are full of carriage return.

Fixed.